server/src/resolvers/Resource.ts

```diff
@@ -162,3 +162,23 @@ export const updateResource: FieldResolver<
     data,
   })
 }
+
+export const resourcesForUser: FieldResolver<
```

Comment:
I think we can get rid of this as discussed

Reply:
We still need to filter the resources though, so you can add the filter on the source.resources resolver